src/hotspot/share/opto/block.cpp

```diff
@@ -1208,6 +1208,22 @@ void PhaseCFG::dump_headers() {
   }
 }
 
+void PhaseCFG::verify_memory_writer_placement(const Block* b, const Node* n) const {
```

Comment:
You could wrap the entire method in `#ifdef ASSERT` and use `NOT_DEBUG_RETURN` in the .hpp (similar to  `verify_strip_mined`).

Reply:
Done, thanks!